lib/model_patches.rb

```diff
@@ -8,6 +8,18 @@
 Rails.configuration.to_prepare do
 
   # Remove UK-specific references to FOI
+  InfoRequest.instance_eval do
+
+    SITE_LAW_USED_READABLE_DATA =
```

Comment:
I'd `s/SITE_LAW_USED_READABLE_DATA/THEME_LAW_USED_READABLE_DATA/`. Its minor but `SITE_` is making me _think_ about what the implication is. The convention elsewhere is to use `theme_BLAH` so I think we should stick with that.